cmd/acra-translator/grpc_api/decryptor.go

```diff
@@ -46,8 +46,44 @@ func NewDecryptGRPCService(data *common.TranslatorData) (*DecryptGRPCService, er
 var (
 	ErrCantDecrypt      = errors.New("can't decrypt data")
 	ErrClientIDRequired = errors.New("clientID is empty")
+	ErrCantEncrypt      = errors.New("can't encrypt data")
 )
 
+// Encrypt encrypt data from gRPC request and returns AcraStruct or error.
+func (service *DecryptGRPCService) Encrypt(ctx context.Context, request *EncryptRequest) (*EncryptResponse, error) {
```

Comment:
I think that `decryptor.go` might be an outdated name here and we'll think about renaming soon